gui/gui/src/TGCommandPlugin.cxx

```diff
@@ -1,6 +1,22 @@
 // @(#)root/gui:$Id$
 // Author: Bertrand Bellenot   26/09/2007
 
+/*************************************************************************
+ * Copyright (C) 1995-2021, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+//////////////////////////////////////////////////////////////////////////
+//                                                                      //
+// TGCommandPlugin                                                      //
+//                                                                      //
+// Class used to redirect command line input/output.                    //
+//                                                                      //
+//////////////////////////////////////////////////////////////////////////
```

Comment:
Why not using the doxygen format:

/** \class TGCommandPlugin
Class used to redirect command line input/output.
**/

The comment will show here: https://root.cern/doc/master/classTGCommandPlugin.html

Reply:
OK, will change it, thanks!

Reply:
Then the other GUI classes might need a similar change. 
But that can be done step by step.

Reply:
Yes, it's what I saw... Same for graf2d 😉